src/client/players.py

```diff
@@ -45,19 +45,25 @@ def isFriend(self, id):
         '''
         Convenience function for other modules to inquire about a user's friendliness.
         '''
+        if id in self._logins:
+            id = self._logins[id].id
```

Comment:
Why not just make this code require IDs and just fix up the slew of ensuing crashes? seems like the quickest way to complete the migration.